src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSetSequence.h

```diff
@@ -228,10 +231,7 @@ class MEDVTKDATAMESHBASE_EXPORT vtkMetaDataSetSequence: public vtkMetaDataSet
   std::vector<vtkMetaDataSet*> MetaDataSetList;
   //ETX
 
-  
  private:
-  
-  vtkMetaDataSetSequence(const vtkMetaDataSetSequence&);      // Not implemented.
   void operator=(const vtkMetaDataSetSequence&);              // Not implemented.
```

Comment:
It's not matters but, with modern c++ to put constructor/destructor/operator not accessible it's:
 void operator=(const vtkMetaDataSetSequence&) = delete; directly into public section.